keyboards/helix/rev3_5rows/keymaps/five_rows/config.h

```diff
@@ -22,15 +22,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifndef CONFIG_USER_H
 #define CONFIG_USER_H
 
-#undef OLED_UPDATE_INTERVAL
-#define OLED_UPDATE_INTERVAL 50
-
 #undef TAPPING_TERM
 #define TAPPING_TERM 300
 #define PERMISSIVE_HOLD
 /* when TAPPING_TERM >= 500 same effect PERMISSIVE_HOLD.
    see tmk_core/common/action_tapping.c */
 
+#undef OLED_UPDATE_INTERVAL
+#define OLED_UPDATE_INTERVAL 50
```

Comment:
If this is using split common, then it should default to 50, per #12107

Reply:
Yes, I know.  However, I prefer to declare it explicitly here. :-)

Reply:
Okay, just making sure. :)